frontend/src/views/styles/modules/_m-legend.scss

```diff
@@ -444,3 +444,18 @@ li[data-id='38'] .item span {
   background-color: transparent !important;
   background-image: url('/images/pastoralPattern.png') !important;
 }
+
+.m-legend-timeline {
+  margin-top: 20px;
+  display: flex;
+  align-items: center;
+
+  .timeline-title {
+    font-size: rem(12px);
+    margin-right: 5px;
+  }
+
+  .timeline {
+    flex: 1;
```

Comment:
I would also add `max-width: 140px` so that the datepicker looks more “inline” like it is on the [design](https://www.figma.com/file/ulN4NKgXHHDncKyIZBeYqV/Resilience-Atlas-%5BInternal%5D?type=design&node-id=623-1561&t=lvdgtloY2mxUJPzX-0).